components/header/src/Header.tsx

```diff
@@ -35,6 +35,8 @@ export type HeaderProps = StandardProps & {
   signOutHref?: string
   /** Sign out link text */
   signOutText?: string
+  /** Logo ReactNode or null */
+  logo?: ReactNode | null
```

Comment:
I think ReactNode already allows `null`.
```suggestion
  /** Custom logo, use null to remove */
  logo?: ReactNode
```